Adyen/Client.cs

```diff
@@ -66,14 +75,30 @@ public Client(string xapikey, Environment environment, string liveEndpointUrlPre
                 LiveEndpointUrlPrefix = liveEndpointUrlPrefix
             };
             SetEnvironment(environment, Config.LiveEndpointUrlPrefix);
-            _client = new HttpClientWrapper(Config, new System.Net.Http.HttpClient());
+            
+            // Set Timeout for HttpClient
+            var httpClient = new System.Net.Http.HttpClient();
+            if (Config.Timeout != default)
+            {
+                httpClient.Timeout = TimeSpan.FromMilliseconds(Config.Timeout);
+            }
+            
+            _client = new HttpClientWrapper(Config, httpClient);
         }
 
         public Client(Config config)
         {
             Config = config;
             SetEnvironment(Config.Environment, Config.LiveEndpointUrlPrefix);
-            _client = new HttpClientWrapper(Config, new System.Net.Http.HttpClient());
+            
+            // Set Timeout for HttpClient

```

Comment:
can we move this code to a helper function rather than copying it three times?

Reply:
Done